frontend/src/concepts/pipelines/apiHooks/mlmd/__tests__/useArtifactsFromMlmdContext.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { testHook, standardUseFetchState } from '~/__tests__/unit/testUtils/hooks';
+import { Context, MetadataStoreServicePromiseClient } from '~/third_party/mlmd';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { useArtifactsFromMlmdContext } from '~/concepts/pipelines/apiHooks/mlmd/useArtifactsFromMlmdContext';
+
+// Mock the usePipelinesAPI hook
+jest.mock('~/concepts/pipelines/context', () => ({
+  usePipelinesAPI: jest.fn(),
+}));
+
+// Mock the MetadataStoreServicePromiseClient
+jest.mock('~/third_party/mlmd', () => {
+  const originalModule = jest.requireActual('~/third_party/mlmd');
+  return {
+    ...originalModule,
+    MetadataStoreServicePromiseClient: jest.fn().mockImplementation(() => ({
+      getArtifactsByContext: jest.fn(),
+    })),
+  };
+});
+
+describe('useArtifactsFromMlmdContext', () => {
+  const mockClient = new MetadataStoreServicePromiseClient('');
+  const mockUsePipelinesAPI = usePipelinesAPI as jest.Mock;
+  const mockGetArtifactsByContext = mockClient.getArtifactsByContext as jest.Mock;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockUsePipelinesAPI.mockReturnValue({
+      metadataStoreServiceClient: mockClient,
+    });
+  });
+
+  it('should fetch and return artifacts', async () => {
+    const mockArtifacts = [{ id: 'artifact1' }, { id: 'artifact2' }];
+    mockGetArtifactsByContext.mockResolvedValue({ getArtifactsList: () => mockArtifacts });
+
+    const renderResult = testHook(useArtifactsFromMlmdContext)(new Context());
+
+    expect(renderResult.result.current).toStrictEqual(standardUseFetchState([]));
```

Comment:
Unnecessary to change but just FYI:
```suggestion
    expect(renderResult).hookToStrictEqual(standardUseFetchState([]));
```